conda_build/cli/main_render.py

```diff
@@ -193,7 +192,6 @@ def parse_args(args: Sequence[str] | None) -> tuple[ArgumentParser, Namespace]:
     return parser, parser.parse_args(args)
 
 
-@deprecated.argument("24.1.1", "24.3.0", "print_results")
```

Comment:
Not sure I follow with this

Reply:
This was a keyword argument that had been removed, the decorator only raises a warning if the argument was passed in